modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -75,50 +75,65 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 $ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
```

Comment:
remove the '/'.    cat /journald.conf  -->  cat journald.conf

Reply:
@anpingli - Done!